depthai_sdk/src/depthai_sdk/oak_device.py

```diff
@@ -21,21 +21,82 @@ def imageSensors(self) -> List[dai.CameraBoardSocket]:
     def info(self) -> dai.DeviceInfo:
         return self.device.getDeviceInfo()
 
-    def initCallbacks(self, pipeline: dai.Pipeline):
+    def stats_report(self) -> Dict[str, Any]:
```

Comment:
This looks very robothub-ish function - can we move such functions into Hub SDK, which can just access device object from OakCamera?

Reply:
Yes, sure. I will move it.